robotoff/insights/importer.py

```diff
@@ -311,8 +343,9 @@ def generate_insights(
             for candidate in candidates:
                 if candidate.automatic_processing is None:
                     logger.warning(
-                        "Insight with automatic_processing=None: %s", candidate.__data__
+                        "Insight with automatic_processing=None", candidate.__data__
```

Comment:
This way candidate.__data__ won't be used !
Either you want to log it and need %s
Either you just want it in sentry and you can add it with `extra={"canditate": candidate.__data__}`

Reply:
Well it did render with '%s' in sentry, that's why. But it's really minor, I revert this.

Reply:
FYI: It's the normal pattern for sentry, and that's for sentry that we use` %s", param`, because it helps sentry group entries.